web/html/src/build/webpack.config.js

```diff
@@ -41,6 +41,11 @@ module.exports = (env, argv) => ([
             }
         ]
     },
+    resolve: {
+      alias: {
+        components: path.resolve(__dirname, '../components/'),
```

Comment:
couldn't we add also `utils` to the list ?

Reply:
sure :) I will do it. 

Even better would be to have a folder "shared" and then have "shared/components" and "shared/utils", but that's too much work to do right now

Reply:
:+1: